NuGet.Config

```diff
@@ -21,6 +21,8 @@
       <package pattern="messagepack.annotations" />
       <package pattern="microsoft.*" />
       <package pattern="moq" />
+      <package pattern="MSTest.TestAdapter" />
+      <package pattern="MSTest.TestFramework" />
```

Comment:
I can't find where these are being used. Can it be removed?

Reply:
Seems no. Pls check this [build ](https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=6305254&view=results):
```
##[error]test\NuGet.Tests.Apex\NuGet.OptProf\NuGet.OptProf.csproj(0,0): Error NU1100: Unable to resolve 'MSTest.TestAdapter (>= 1.2.0)' for '.NETFramework,Version=v4.7.2'. PackageSourceMapping is enabled, the following source(s) were not considered: dotnet-eng, dotnet-public, myget-legacy@Local, nuget-build, vside_vs-impl, vside_vssdk.
D:\a\_work\1\s\test\NuGet.Tests.Apex\NuGet.OptProf\NuGet.OptProf.csproj : error NU1100: Unable to resolve 'MSTest.TestAdapter (>= 1.2.0)' for '.NETFramework,Version=v4.7.2'. PackageSourceMapping is enabled, the following source(s) were not considered: dotnet-eng, dotnet-public, myget-legacy@Local, nuget-build, vside_vs-impl, vside_vssdk. [D:\a\_work\1\s\build\restorehelper.targets]
##[error]test\NuGet.Tests.Apex\NuGet.OptProf\NuGet.OptProf.csproj(0,0): Error NU1100: Unable to resolve 'MSTest.TestFramework (>= 1.2.0)' for '.NETFramework,Version=v4.7.2'. PackageSourceMapping is enabled, the following source(s) were not considered: dotnet-eng, dotnet-public, myget-legacy@Local, nuget-build, vside_vs-impl, vside_vssdk.
D:\a\_work\1\s\test\NuGet.Tests.Apex\NuGet.OptProf\NuGet.OptProf.csproj : error NU1100: Unable to resolve 'MSTest.TestFramework (>= 1.2.0)' for '.NETFramework,Version=v4.7.2'. PackageSourceMapping is enabled, the following source(s) were not considered: dotnet-eng, dotnet-public, myget-legacy@Local, nuget-build, vside_vs-impl, vside_vssdk. [D:\a\_work\1\s\build\restorehelper.targets]
##[error]test\NuGet.Tests.Apex\NuGet.Tests.Apex\NuGet.Tests.Apex.csproj(0,0): Error NU1100: Unable to resolve 'MSTest.TestAdapter (>= 1.2.0)' for '.NETFramework,Version=v4.7.2'. PackageSourceMapping is enabled, the following source(s) were not considered: dotnet-eng, dotnet-public, myget-legacy@Local, nuget-build, vside_vs-impl, vside_vssdk.
D:\a\_work\1\s\test\NuGet.Tests.Apex\NuGet.Tests.Apex\NuGet.Tests.Apex.csproj : error NU1100: Unable to resolve 'MSTest.TestAdapter (>= 1.2.0)' for '.NETFramework,Version=v4.7.2'. PackageSourceMapping is enabled, the following source(s) were not considered: dotnet-eng, dotnet-public, myget-legacy@Local, nuget-build, vside_vs-impl, vside_vssdk. [D:\a\_work\1\s\build\restorehelper.targets]
##[error]test\NuGet.Tests.Apex\NuGet.Tests.Apex\NuGet.Tests.Apex.csproj(0,0): Error NU1100: Unable to resolve 'MSTest.TestFramework (>= 1.2.0)' for '.NETFramework,Version=v4.7.2'. PackageSourceMapping is enabled, the following source(s) were not considered: dotnet-eng, dotnet-public, myget-legacy@Local, nuget-build, vside_vs-impl, vside_vssdk.
D:\a\_work\1\s\test\NuGet.Tests.Apex\NuGet.Tests.Apex\NuGet.Tests.Apex.csproj : error NU1100: Unable to resolve 'MSTest.TestFramework (>= 1.2.0)' for '.NETFramework,Version=v4.7.2'. PackageSourceMapping is enabled, the following source(s) were not considered: dotnet-eng, dotnet-public, myget-legacy@Local, nuget-build, vside_vs-impl, vside_vssdk. [D:\a\_work\1\s\build\restorehelper.targets]
##[error]test\NuGet.Tests.Apex\NuGet.Tests.Apex\NuGet.Tests.Apex.csproj(0,0): Error NU1100: Unable to resolve 'MSTest.TestAdapter (>= 1.2.0)' for '.NETFramework,Version=v4.7.2/x86'. PackageSourceMapping is enabled, the following source(s) were not considered: dotnet-eng, dotnet-public, myget-legacy@Local, nuget-build, vside_vs-impl, vside_vssdk.
D:\a\_work\1\s\test\NuGet.Tests.Apex\NuGet.Tests.Apex\NuGet.Tests.Apex.csproj : error NU1100: Unable to resolve 'MSTest.TestAdapter (>= 1.2.0)' for '.NETFramework,Version=v4.7.2/x86'. PackageSourceMapping is enabled, the following source(s) were not considered: dotnet-eng, dotnet-public, myget-legacy@Local, nuget-build, vside_vs-impl, vside_vssdk. [D:\a\_work\1\s\build\restorehelper.targets]
##[error]test\NuGet.Tests.Apex\NuGet.Tests.Apex\NuGet.Tests.Apex.csproj(0,0): Error NU1100: Unable to resolve 'MSTest.TestFramework (>= 1.2.0)' for '.NETFramework,Version=v4.7.2/x86'. PackageSourceMapping is enabled, the following source(s) were not considered: dotnet-eng, dotnet-public, myget-legacy@Local, nuget-build, vside_vs-impl, vside_vssdk.
D:\a\_work\1\s\test\NuGet.Tests.Apex\NuGet.Tests.Apex\NuGet.Tests.Apex.csproj : error NU1100: Unable to resolve 'MSTest.TestFramework (>= 1.2.0)' for '.NETFramework,Version=v4.7.2/x86'. PackageSourceMapping is enabled, the following source(s) were not considered: dotnet-eng, dotnet-public, myget-legacy@Local, nuget-build, vside_vs-impl, vside_vssdk. [D:\a\_work\1\s\build\restorehelper.targets]
  Failed to restore D:\a\_work\1\s\test\NuGet.Tests.Apex\NuGet.OptProf\NuGet.OptProf.csproj (in 1.84 sec).
  Failed to restore D:\a\_work\1\s\test\NuGet.Tests.Apex\NuGet.Tests.Apex\NuGet.Tests.Apex.csproj (in 2.01 sec).
```

Reply:
Oops, sorry. When I first reviewed this, I thought it was `bootstrap.proj`, not `nuget.config`.  Makes sense.